### ISSUE TITLE
[bug] Error when use EMSDK because don't have clang 16 in conan

### ISSUE THREAD

**Issue by paulocoutinhox** [conan-io/conan#11774]
Hi,

Im trying use EMSDK version 3.1.18, but conan don't have clang 16:

```
ERROR: Invalid setting '16' is not a valid 'settings.compiler.version' value.
Possible values are ['3.3', '3.4', '3.5', '3.6', '3.7', '3.8', '3.9', '4.0', '5.0', '6.0', '7.0', '7.1', '8', '9', '10', '11', '12', '13', '14', '15']
Read "http://docs.conan.io/en/latest/faq/troubleshooting.html#error-invalid-setting"
```

Can conan team add version 16?

**Comment by czoido** [comment]
Hi @paulocoutinhox,

Yes, we can add that [version](https://github.com/emscripten-core/emscripten/blob/main/ChangeLog.md#3118---08012022) to the settings.

Thanks a lot for reporting.

**Comment by paulocoutinhox** [comment]
Hi,

I made a PR:
https://github.com/conan-io/conan/pull/11780

Thanks.

**Comment by franramirez688** [comment]
Closed by https://github.com/conan-io/conan/pull/11780

Thanks for the contribution @paulocoutinhox! The fix will be released in the next Conan 1.52 version.
